backend/src/routes/qrCode.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express'
+import { verifyApiKey, verifyFirebaseToken } from '../middlewares/auth'
+import { createQrCode, deleteQrCode, getQrCodes, updateQrCode } from '../controllers/qrCode'
+
+const router = express.Router()
+
+router.get('/', verifyApiKey, verifyFirebaseToken, getQrCodes)
+router.post('/create', verifyApiKey, verifyFirebaseToken, createQrCode)
+router.post('/update', verifyApiKey, verifyFirebaseToken, updateQrCode)
+router.delete('/', verifyApiKey, verifyFirebaseToken, deleteQrCode)
```

Comment:
instead of a delete request make it a post request with the route being `/delete`. I know they will get confused on the frontend side on how to make a delete request.